CONTRIBUTING.md

```diff
@@ -0,0 +1,90 @@
+# Contributing to Newsboat
+
+Thank you for your interest in contributing to Newsboat! Here are a few ways
+you can help:
+
+1. [Feature Requests](#feature-requests)
+2. [Bug Reports](#bug-reports)
+3. [Patches / Pull Requests](#patches--pull-requests)
```

Comment:
I believe there should not be any spaces around the slash. Down below in the section heading, too. The anchor target probably needs to be adjusted as well.